.github/workflows/release-azure-pipelines.yml

```diff
@@ -30,25 +30,28 @@ parameters:
   
 variables:
 - name: tags
-  value: "nonproduction"
+  value: "production"
   readonly: true
 - name: pythonVersion
   value: 3.10
   readonly: true
+- name: artifactsPath
+  value: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}
+  readonly: true
 
 trigger: none
 
 pr: none
 
 resources:
   repositories:
-  - repository: CustomPipelineTemplates
+  - repository: OfficePipelineTemplates
     type: git
-    name: 1ESPipelineTemplates/OfficePipelineTemplates
-    ref: refs/tags/release
+    name: OE/OfficePipelineTemplates

```

Comment:
Interesting! This is the first I've seen this path used. Do you have a link to documentation that references this project/repo?

Reply:
Lot of OE projects are using this but I am not able to find the link (pinging the link to oe projects async). The doc does mention `1ESPipelineTemplates/OfficePipelineTemplates` in its example. I think OE reference might be an old one. Let me search support channel to find any relevant information and if the new way is using `1ESPipelineTemplates/OfficePipelineTemplates`, I can revert this

Reply:
Just checking if you had found anything in the support channel!

Reply:
Yes! Updated this to `1ESPipelineTemplates/OfficePipelineTemplates`. From what I found; it seems both repositories are kept up to date but `1ESPipelineTemplates/OfficePipelineTemplates` seems to be the latest way to do this so going with that,